packages/server/src/service/signup.ts

```diff
@@ -14,8 +14,12 @@ export const sendAndStoreSignup = async (
   info: StateInfo,
   method: ContactMethod,
   id: string,
-  resendReasonAndOriginalDate?: string,
+  options: {
```

Comment:
Nit -- I would make the type separate
```ts
interface Options {
  ...
}
```

so that you can write `{ resendReasonAndOriginalDate, noUpload }: Options` here in the function arguement.